src/app/fdctl/run/tiles/fd_gossip.c

```diff
@@ -317,15 +317,23 @@ gossip_deliver_fun( fd_crds_data_t * data, void * arg ) {
       return;
     }
 
-    uchar * vote_txn_msg = fd_chunk_to_laddr( ctx->dedup_out_mem, ctx->dedup_out_chunk );
+    uchar * vote_txn_msg = fd_chunk_to_laddr( ctx->verify_out_mem, ctx->verify_out_chunk );
     ulong vote_txn_sz    = gossip_vote->txn.raw_sz;
     memcpy( vote_txn_msg, gossip_vote->txn.raw, vote_txn_sz );
 
+    /* DEFENSE-IN-DEPTH: While the underlying txn message is technically verifed
```

Comment:
This comment doesn't seem to make sense. You've only verified one signature with the CRDS message? And it might be for bytes that aren't the actual transaction bytes. Also what does the TODO mean? What impact would it have on "verify traffic" ?

Reply:
There is no performance impact of sending gossiped votes to sigverify, it's not a material volume, and you need to verify them... verifying the CRDS is not the same as verifying the transaction.